front/src/common/markdowneditor/markdowneditor.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { basicSetup, EditorView } from 'codemirror'
+import { EditorState } from '@codemirror/state'
+import { useEffect, useRef } from 'react'
```

Comment:
You already have the React import in the line 1, this line is not necessary

Reply:
Done